README.md

```diff
@@ -66,9 +72,31 @@ optional arguments:
   -v X, --verbose X, --verbosity X
                         Set the verbosity level of the console (0=quiet,
                         1=default, 2=verbose)
+  --otns OtnsServer,
+                        Set the OTNS server address to send OTNS messages to
+```
+
+There is an example of test run script `silk_run_test.py` under unit_tests folder.
+
+### Using OTNS
+[OpenThread Network Simulator](https://github.com/openthread/ot-ns) is a Thread network visualization and management tool.
+Using this tool the topology and messages of a network in Silk's control could be visualized. Run the following to install OTNS.
+Note that it requires Go which can be installed from https://golang.org/dl/, after which `$(go env GOPATH)/bin` should be added
+to `$PATH`.
+
+```shell
+git clone https://github.com/openthread/ot-ns.git ./otns
+cd otns
+./script/install-deps
+./script/install
 ```
 
-There is an example of test run script silk_run_test.py under unit_tests folder.
+Then follow these steps to use Silk with OTNS:
+
+1. Configure `ClusterID` for the cluster and `OTNSVisPosition` for each node in `hwconfig.ini`.
```

Comment:
Please specify how much range is allowed in the OTNSVisPosition parameters here
e.g. if 1 dev board has x, y coordinates as OTNSVisPosition: 200, 200 for 2nd nordic board is OTNSVisPosition: 250, 250 good enough or it is too close and what will be too far. Thus providing a range for position in README will help. Thanks 
[Dev-9BE1]
HwModel: nRF52840_OpenThread_Device
HwRev: 1.0
InterfaceSerialNumber: F43C28989BE1
USBInterfaceNumber: 1
DutSerial: 683253620
OTNSVisPosition: 200, 200

[Dev-1EA7]
HwModel: nRF52840_OpenThread_Device
HwRev: 1.0
InterfaceSerialNumber: ECDADC3D1EA7
USBInterfaceNumber: 1
DutSerial: 683247053
OTNSVisPosition: 250, 250